src/IotHub/IotHub/IotHub.csproj

```diff
@@ -11,6 +11,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.4.3" />
```

Comment:
These libraries add in dependencies that will likely not work in PowerShell (multiple version of Newtonsoft.Json referenced).  This will require some analysis of the dependencies, and potentially some modifications to the libraries or other workarounds.

Note that this is the stable version of your module, so you should not take a dependency on a preview library unless you know that the API surface will not be broken.

Reply:
removed preview library and updated to the stable library which is using same NewtonSoft.Json dependency(removed multiple version of newtonSoft.Json)